CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -303,6 +303,32 @@ public function upgrade_4_7_16($rev) {
     $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
   }
 
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_4_7_17($rev) {
+    $this->addTask('CRM-19385 - Look at REMOVING id column from cache tables', 'removeIdFromCacheTables');
```

Comment:
Also this comment should be "Remove id column", not "Look at REMOVING id column"